downstream/modules/platform/ref-controller-set-up-jump-host.adoc

```diff
@@ -1,29 +1,70 @@
 [id="ref-controller-set-up-jump-host"]
 
-= Set up a jump host to use with {ControllerName}
+= Configuring {ControllerName} to use jump hosts connecting to managed nodes
 
 Credentials supplied by {ControllerName} do not flow to the jump host through ProxyCommand. 
 They are only used for the end-node when the tunneled connection is set up.
 
-You can configure a fixed user/keyfile in the AWX user's SSH configuration in the ProxyCommand definition that sets up the connection through the jump host. 
+[discrete]
+== Configure a fixed user/keyfile in your SSH configuration file
 
-For example:
+You can configure a fixed user/keyfile in your SSH configuration file in the ProxyCommand definition that sets up the connection through the jump host. 
+
+.Prerequisites
+* Check whether all jump hosts are reachable from any node that establishes a SSH connection to the managed nodes, such as a Hybrid Controller or an Execution Node.
+
+.Procedure
+. Create a SSH configuration file `/var/lib/awx .ssh/config` on each node with the following details
 
 [literal, options="nowrap" subs="+attributes"]
 ----
-Host tampa
-Hostname 10.100.100.11
-IdentityFile [privatekeyfile]
+Host jumphost.example.com
+    Hostname jumphost.example.com
+    User <jumphostuser>
+    Port <jumphostport>
+    IdentityFile ~.ssh/id_rsa
+    StrictHostKeyChecking no
+    ProxyCommand ssh -W %h:%p jumphost.example.com
+----
 
-Host 10.100..
-Proxycommand ssh -W [jumphostuser]@%h:%p tampa
+* The code specifies the configuration required to connect to the jum host 'jumphost.example.com'
+* {ControllerNameStart} establishes a SSH connection from each node to the managed nodes.
+* Example values `jumphost.example.com`, `jumphostuser`, `jumphostport` and `~/.ssh/id_rsa` must be changed according to your environment
+* Add a Host matching block to the already created SSH configuration file `/var/lib/awx/.ssh/config`` on the node, for example:
++ 
+[literal, options="nowrap" subs="+attributes"]
+----
+Host 192.0.* 
```

Comment:
Might want to put a `...` in here. This isn't a complete configuration, it's trying to highlight that you can use `Host <regex>` to limit the set of hosts you want to use a jump host for.

Personally, I would remove the `ProxyCommand ssh -W %h:%p jumphost.example.com` line to reduce confusion

```
Host 192.0.*
    ...
```